demo-transactions/src/main/kotlin/io/holixon/cqrshexagonaldemo/demoparent/transactions/command/domain/model/event/MoneyWithdrawnEvent.kt

```diff
@@ -0,0 +1,6 @@
+package io.holixon.cqrshexagonaldemo.demoparent.transactions.command.domain.model.event
+
+import io.holixon.cqrshexagonaldemo.demoparent.transactions.command.domain.model.account.Iban
+import io.holixon.cqrshexagonaldemo.demoparent.transactions.command.domain.model.account.Money
+
+data class MoneyWithdrawnEvent(val iban: Iban, val balance: Money, val timeStamp: Long) : Event
```

Comment:
We shouldn't use `Long` here, only domain objects should be used as parameter for container classes. If it is needed, a `Timestamp` object should be created.